lib/Doctrine/deploy/AddServiceGroups.php

```diff
@@ -10,6 +10,15 @@
 $stFileName = __DIR__ . "/" . $GLOBALS['dataDir'] . "/ServiceGroups.xml";
 $sts = simplexml_load_file($stFileName);
 
+// Checking the XML file has parsed correctly
+if ($sts === false) {
+    echo "There were errors parsing the XML file.\n";
+    foreach (libxml_get_errors() as $error) {
+        echo $error->message;
```

Comment:
Maybe add a comment nothing that `$error` will be a libXMLError object?

Reply:
@rowan04 could you add this comment?

Reply:
ok

Reply:
done